--- a/docs/Makefile.in	2022-06-13 11:44:59.000000000 +1000
+++ b/docs/Makefile.in	2022-06-13 11:45:40.000000000 +1000
@@ -1175,7 +1175,7 @@
 gih: gnuplot.gih
 
 gnuplot.gih: doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc
-	$(AM_V_GEN)./doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc gnuplot.gih
+	$(srcdir)/../build-native/docs/doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc gnuplot.gih
 
 # To include all terminals in the .gih file
 allgih: alldoc2gih$(EXEEXT) $(srcdir)/gnuplot.doc
