--- a/docs/Makefile.in	2025-12-16 21:52:03
+++ b/docs/Makefile.in	2026-04-03 23:51:25
@@ -1198,7 +1198,7 @@
 gih: gnuplot.gih
 
 gnuplot.gih: doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc
-	$(AM_V_GEN)./doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc gnuplot.gih
+	$(srcdir)/../build-native/docs/doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc gnuplot.gih
 
 # To include all terminals in the .gih file
 allgih: alldoc2gih$(EXEEXT) $(srcdir)/gnuplot.doc
--- a/demo/Makefile.in	2025-12-16 21:52:03
+++ b/demo/Makefile.in	2026-04-04 00:02:13
@@ -722,10 +722,10 @@
 
 all: $(BINARY_FILES)
 
-$(BINARY_FILES): ../src/bf_test $(top_srcdir)/demo/dodecahedron.dat
+$(BINARY_FILES): $(srcdir)/../build-native/src/bf_test $(top_srcdir)/demo/dodecahedron.dat
 	@echo Creating binary data files
-	test "$(top_srcdir)" = "$(top_builddir)" || cp $(top_srcdir)/demo/dodecahedron.dat .
-	@../src/bf_test
+	test "$(top_srcdir)" = "$(top_builddir)" || cp -f $(top_srcdir)/demo/dodecahedron.dat .
+	@$(srcdir)/../build-native/src/bf_test
 	test "$(top_srcdir)" = "$(top_builddir)" || rm -f ./dodecahedron.dat
 
 check-prepare:
