diff --git a/docs/Makefile.am b/docs/Makefile.am
index b20918ab0..7f831f115 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -268,7 +268,7 @@ doc2hlp_SOURCES = doc2hlp.c termdoc.c
 gih: gnuplot.gih
 
 gnuplot.gih: doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc
-	$(AM_V_GEN)./doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc gnuplot.gih
+	$(srcdir)/../build-native/docs/lib/doc2gih$(EXEEXT) $(srcdir)/gnuplot.doc gnuplot.gih
 
 doc2gih_SOURCES = doc2gih.c termdoc.c
 
